.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contribute to PurotoApp/Berry
+
+First of all, thank you for taking a moment to consider contributing to Puroto! You're awesome.
+
+> Before contributing, please read our [Code of Conduct](https://github.com/PurotoApp/Berry/blob/master/.github/CODE_OF_CONDUCT.md) first. We want this community to stay civil and respecting to each other.
```

Comment:
On `We want this community to stay civil and respecting to each other.`, the word `respecting` fits weirdly in this context. it will sound better if you replace it with `respectful`